modules/testing/src/backend/builder.ts

```diff
@@ -0,0 +1,90 @@
+import * as sdk from 'botpress/sdk'
+import _ from 'lodash'
+
+import { Scenario } from './typings'
+import { convertLastMessages } from './utils'
+
+export class Builder {
```

Comment:
Code is good I just don't get why you wrote a class for this instead of a simple set of stateless functions